templates/quarkus/events/src/main/resources/application.properties

```diff
@@ -1,3 +1,8 @@
 # Configuration file
 # key = value
-quarkus.funqy.export=echo
\ No newline at end of file
+
+quarkus.funqy.export=echo
+
+quarkus.smallrye-health.readiness-path=/readiness
+quarkus.smallrye-health.liveness-path=/liveness
```

Comment:
@lance I had to add this, otherwise path is `ready/live` instead of `readiness/liveness`.

Same below.

Reply:
@matejvasek is the full path `/health/readiness` or just `/readiness`? I kind of like having the `/health` prefix as it minimizes the pollution at the root.

Reply:
The paths are `/health/readiness` and `readiness/liveness`.
There is property `quarkus.smallrye-health.root-path` which is by default "/health".